.github/workflows/cd.yml

```
@@ -53,4 +68,4 @@ jobs:
           SLACK_CHANNEL: java-sync
         uses: Ilshidur/action-slack@master
         with:
-          args: 'Version: ${{ steps.vars.outputs.tag }} of the library is released to bintray'
+          args: 'Version: <a href="https://github.com/commercetools/commercetools-sync-java/releases/tag/${{ steps.vars.outputs.tag }}">${{ steps.vars.outputs.tag }}</a> of the library is released to bintray'
```

```suggestion
          args: 'Version: <a href="https://github.com/commercetools/commercetools-sync-java/releases/tag/${{ steps.vars.outputs.tag }}">${{ steps.vars.outputs.tag }}</a> of the library is published to <a href="https://bintray.com/commercetools/maven/commercetools-sync-java"> bintray repository</a>, check <a href="https://github.com/commercetools/commercetools-sync-java/blob/master/docs/BUILD.md#step-2-publish">here</a> for the next steps.'
```

This might be confusing for the new people, I also added the link of repo + next steps for a complete release. 
WDYT ?

---

39dd183c5